libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -822,6 +825,22 @@ string YulUtilFunctions::arrayAllocationSizeFunction(ArrayType const& _type)
 	});
 }
 
+string YulUtilFunctions::structAllocationSizeFunction(StructType const& _type)
```

Comment:
Is this really worth a function :-)? It's always just a literal, isn't it?

Reply:
Yep, removed it